docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,439 @@
+
+**SYNOPSIS**  
+`pbench-register-tool-trigger[-C][-g][--start-trigger][--stop-trigger][--help]`
+
+**DESCRIPTION**  
+Registers tool with the given group and start and stop tool trigger text
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+-g, --group,--groups STR  
+Registers the trigger in the <group-name> group. If no group is specified, the 'default'
+
+--start-trigger STR  
+[required]
+
+--stop-trigger STR  
+[required]
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-results-move
+
+---
+
+**NAME**  
+pbench-results-move, Move results tarball to the server
+
+**SYNOPSIS**  
+`pbench-results-move[-C][--controller][--token][--delete][--xz-single-threaded][--show-server][--help]
+`
+
+**DESCRIPTION**  
+Move result directories to the configured Pbench server.This command is used when pushing result tarball using token.
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable,if defined) [required]
+
+--controller STR  
+Override the default controller name
+
+--token STR  
+pbench server authentication token [required]
+
+--delete / --no-delete  
+Remove local data after successful copy [default: delete]
+
+--xz-single-threaded  
+Use single-threaded compression with 'xz'
+
+--show-server STR  
+Display information about the pbench server where the result(s) will be moved (Not implemented)
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-user-benchmark
+
+---
+
+**NAME**  
+pbench-user-benchmark, Control start/stop/post-process-tools.
+
+**SYNOPSIS**  
+`pbench-user-benchmark[-C][--tool-group][--iteration-list][--sysinfo][--pbench-pre][--pbench-post][--use-tool-triggers][--no-stderr-capture]-- <script to run>`
+
+**DESCRIPTION**  
+Collects data from the tools registered. Here are the steps involved
+
+- Invoking pbench-user-benchmark with your workload generator as an argument: that will start the collection tools on all the hosts
+- Next, it will run your workload generator; when that finishes, it will stop the collection tools on all the hosts
+- Finally, the postprocessing phase will gather the data from all the remote hosts and run the postprocessing tools on everything.
+
+**OPTIONS**  
+-C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable,if defined) [required]
+
+--tool-group  
+The tool group to use for the list of tools
+
+--iteration-list  
+A file containing a list of iterations to run for the provided script;
+the file should contain one iteration per line, with a leading '#' (hash) character used for comments, blank lines are ignored; each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script;  
```

Comment:
I'd move ", with a leading '#' (hash) character used for comments, blank lines are ignored" to a separate sentence after this. It's confusing in the middle of a long phrase. Describe the meaning of the lines, and then add that comments and blank lines are ignored.